package-lock.json

```diff
@@ -11581,7 +11581,6 @@
         "inquirer": "^6.2.2",
         "lodash": "^4.17.14",
         "needle": "^2.2.4",
-        "opn": "^5.5.0",
```

Comment:
You cannot remove a dependency of an external package; Undo this line removal.

[`snyk`](https://www.npmjs.com/package/snyk) appears to be referenced through [`dispensary`](https://www.npmjs.com/package/dispensary) (as an entry in [`devDependencies`](https://github.com/mozilla/dispensary/blob/da269f902ca456d1402623a40370462bb791b1df/package.json#L54)). This should therefore not affect the `web-ext` package when you install `web-ext`.

If you want to remove this `opn` dependency, open a pull request at snyk - https://github.com/snyk/snyk
, and once a version with the updated package has been published, open a pull request at dispensary - https://github.com/mozilla/dispensary
.